dev/NavigationView/NavigationView.cpp

```diff
@@ -1483,15 +1485,34 @@ void NavigationView::UpdatePaneLayout()
         const auto totalAvailableHeight = [this]() {
             if (const auto &paneContentRow = m_itemsContainerRow.get())
             {
-                // 20px is the padding between the two item lists
-                if (const auto &paneFooter = m_leftNavFooterContentBorder.get())
+                const double itemsContainerMargin = [this]() {
+                    if (const auto itemsContainer = m_itemsContainer.get())
+                    {
+                        auto const margin = itemsContainer.Margin();
+                        return margin.Top + margin.Bottom;
+                    }
+                    return 0.0;
+                }();
+                auto availableHeight = paneContentRow.ActualHeight() - itemsContainerMargin;
+
+                if (PaneFooter())
+                {
+                    availableHeight -= 21;
```

Comment:
Comment on what this value means or add a static constexpr?

Reply:
Added a comment, good idea.